cmd/webhook/main.go

```diff
@@ -329,6 +333,14 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 					sourcesv1alpha2_: &sourcesv1alpha2.SinkBinding{},
 				},
 			},
+			sourcesv1beta1.Kind("SinkBinding"): {
+				DefinitionName: sources.SinkBindingResource.String(),
+				HubVersion:     sourcesv1alpha2_,
+				Zygotes: map[string]conversion.ConvertibleObject{
+					sourcesv1alpha2_: &sourcesv1alpha2.SinkBinding{},
+					sourcesv1beta1_:  &sourcesv1beta1.SinkBinding{},
```

Comment:
@vaikas question here:
we have v1alpha1, v1alpha2 and v1beta1 now...
how this is supposed to work.. v1alpha1 is converted to v1alpha2, 
and then v1alpha2 to v1beta1?

Or should I add another entry in this map (or the one with sourcesv1alpha2) and add another case in the switch statements in the conversion files?

Reply:
Well, I guess it sort of depends on what the plan here is. If we're planning on serving all three of them, then that seems to me that we should have the following 'logical' conversions in place:
v1alpha1<->v1alpha2<->v1beta1
So, I guess I'd actually expect the hubversion to be v1alpha1. But, whatever version you have for the hub, that's the version that knows how to convert back and forth between all the revisions. So, if you have hub as v1alpha2, then it should support conversions to/from v1alpha1 as well as v2alpha1.
So, this map should have all three versions, and then yes, you need to add a conversions to *_conversion.go files. For example, using serving.Configuration (which has v1alpha1, v1beta1, and v1), it looks like this:
https://github.com/knative/serving/blob/master/cmd/webhook/main.go#L192

And the conversion looks like so:
https://github.com/knative/serving/blob/master/pkg/apis/serving/v1alpha1/configuration_conversion.go#L28

Reply:
Thanks Ville! Much clearer, will look at what serving is doing and update this PR. 
Will ping you once is ready for review.